src/pages/verify/e-ids/frejaid.mdx

```diff
@@ -0,0 +1,182 @@
+
+## Testing
+FrejaID is available to anyone, but some features are only available for Swedish citizens and EEA citizens with a coordination number (samordningsnummer) issued by the Swedish Tax Authority.
+For testing, you can easily generate a test user with your chosen level of access (which FrejaID calls the "registration level").
+
+### Installing the Freja app
+You can use the regular Freja app for testing.
+Simply download the app from the [App Store](https://apps.apple.com/us/app/freja/id1256552092) or [Google Play](https://play.google.com/store/apps/details?id=com.verisec.mobile.frejaeid).
+
+#### Configuring the app for testing
+To use the Freja app for testing, you must activate test mode.
+Note that the app will stay in test mode until you delete it from your device, so you should not use your normal Freja app for testing.
+
+To activate test mode in the Freja app:
+1. Open the Freja app
+2. Press "New User"
+3. Press "Get Started"
+4. Toggle the "General Terms & Conditions" button six times
+5. You should now see a screen saying "Welcome to Test Mode"
+6. Press "Start Test Mode"
+7. Press "OK" in the pop-up warning
+8. The Freja app will now close
+
+To create a Basic test user for the Freja app in test mode:
+1. Open the Freja app
+2. Press "New User"
+3. Press "Get Started"
+4. Accept the "General Terms & Conditions", the "Privacy Policy" and the disclaimer about falsifying identities
+5. Press "Continue"
+6. Press "Continue"
+7. Press "Set Notifications" if you want to test push Notifications
+8. Press "Allow" in the notification popup from your device
+9. Enter your email address (it must be a real email address where you can receive emails)
+10. Confirm your email address by pressing "Send"
+11. Open your email account and press the "Confirm" link in the email from "Freja - TEST MODE"
+12. Open the Freja app
+13. Choose a PIN code and enter it in both fields
+14. Press "Confirm"
+15. Your device may ask you whether you also want to register biometrics such as FaceID or fingerprint for Freja - select "Allow" if you want to use it
+16. Press "OK"
+17. Press "Later" when asked if you want to upgrade your account
+You now have a Basic Freja test user.
+
+To upgrade your Basic test user to Freja Extended in test mode:
+1. Open a browser and go to the [Freja Test Vetting Portal](https://vettingportal.test.frejaeid.com/)
+2. Enter the email you used to register the test user
+3. Press "Upgrade"
+4. Choose "Sweden" if you want access to test all features of Freja (and if you want to upgrade to Freja+ later)
+5. Press "Next"
+7. Select in a first name and a last name as well as a birth date and gender for the test user
+8. Press "Generate" to obtain a personal identity number (social security number) for the test user
+9. Write down the generated personal identity number - you will not be able to see it later
+9. Press "Upgrade"
+Your test user will now be registered as Extended, which gives you access to testing with more scopes.
```

Comment:
The text is displayed on one line here. Maybe adding a colon after "Press "Upgrade"" is an option. 
![Screenshot 2025-04-24 at 16 39 02](https://github.com/user-attachments/assets/97cb061e-26b7-44fa-9f9d-f9e6c353e91c)

Reply:
Fixed and added some more headers to make it easier to navigate in b6881d2.